src/pages/components/RevokeActionsButton.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Button } from '@digicatapult/ui-component-library'
+
+const reasonsDummyJSON = {
+  predefinedReasons: {
+    0: { selection: null },
+    1: { selection: null },
+    2: { selection: null },
+  },
+  otherReason: '',
+}
+
+export default function RevokeActionsButton({
+  handleRevoke,
+  disabled,
+  revoking,
+}) {
+  const onClick = () => {
+    const answer = window.confirm('Reason for revoking?')
+    if (answer) {
+      handleRevoke(reasonsDummyJSON)
+    }
+  }
+
+  return (
+    <LargeButton
+      onClick={onClick}
+      disabled={disabled}
+      variant="roundedPronounced"
+    >
+      Revoke {revoking && '...'}
```

Comment:
Could you animate the dots like when we submit a certificate? At the moment the static dots aren't very appealing